From 5ba0df319351e42065b742c24315f3cf9264c590 Mon Sep 17 00:00:00 2001
From: weidong <weidong@uniontech.com>
Date: Thu, 15 Oct 2020 18:34:47 +0800
Subject: [PATCH] Fix compilation errors

---
 introduction.pro | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/introduction.pro b/introduction.pro
index 39d97c1..ed10cdf 100644
--- a/introduction.pro
+++ b/introduction.pro
@@ -71,14 +71,10 @@ SOURCES += \
 RESOURCES += \
     dde-introduction.qrc
 
-contains(TARGET_ARCH, x86_64) {
-
-}else{
     DEFINES += DISABLE_VIDEO
     HEADERS -= src/modules/videowidget.h
     SOURCES -= src/modules/videowidget.cpp
-    # PKGCONFIG -= libdmr
-}
+    PKGCONFIG -= libdmr
 
 
 desktop.path = $$PREFIX/share/applications/
-- 
2.18.4

